### ISSUE TITLE
Use Recorder db_url for database location

### ISSUE THREAD

**Issue by mniquette** [hassio-addons/addon-sqlite-web#199]
# Problem/Motivation 

> I would like to use this tool but my database is not located in the config directory. 
## Expected behavior

> The addon would do some simple searching for the database rather than just assuming it is located in the config directory.
## Actual behavior

> It just creates a new blank database file in the config directory.

## Steps to reproduce

> Described in original problem statement.

## Proposed changes

> The Recorder allows the database location to be specified. So utilize the db_url to determine the database location. This would retain the simple setup philosophy of this tool.

**Comment by Shalobee** [comment]
I have the exact same problem and this solution sounds like a good fix.

**Comment by mniquette** [comment]
Thanks for considering this request. My db is quite large and so I backup it up on a less regular basis than the other files in my system. Since the backup mechanisms I've used (system, google, samba) only allow directory exclusion the obvious answer is to place the db in a folder other than config (I use the share folder). Also my Recorder spec is in a separate package file (system.yaml) which you indicated may be the case. If the requirement was that the db be either in config or some other specific folder(s) or the Recorder spec needed to be in the configuration.yaml I could work with that too.

**Comment by pwgdrx** [comment]
I also moved the database to a different path so as not to back it up.
However, no setting for a different path is provided.

**Comment by michaelblight** [comment]
While I can understand that the add-on cannot automatically find the database if elsewhere, can we have a means of specifying its url? Some time ago I switched to using MariaDB, so it would be nice to use this add-on to explore the data. I want to review how much data is being stored by the recorder for each entity as well as the long-term statistics, with a view to possibly optimizing the database size without the sledgehammer of `purge_keep_days` (which I have set to 5).
